wiki/Beatmap_Discussion/en.md

```diff
@@ -111,13 +113,17 @@ This is where you type your mod. If you use the Timeline section, you must inclu
 
 ![](img/similar-post-warning.jpg)
 
-If your timestamp submitted through the Timeline section is close to another mod already posted, you will have to confirm that the mod doesn't address the issue you are currently submitting. Please check the discussion field before checking the box!
+If your timestamp submitted through the Timeline section is close to another mod already posted, you will have to confirm that the mod doesn't address the issue you are currently submitting. **Please check the discussion field before checking the box!**
 
 ### Mod type
 
 ![](img/mod-type.jpg)
 
-This is where you submit your mod. You can choose between Praise, Suggestion, or Problem. Each is fairly self-explanatory. Clicking on this will submit your mod. If you made any mistake or need to make an edit to your mod, find the post in the Discussion section.. You can add Hype to the beatmap by submitting Praise in the General (All Difficulties) section.
+This is where you submit your mod. You can choose between Praise, Suggestion, or Problem. Each is fairly self-explanatory. **Clicking on this will submit your mod. If you made any mistake or need to make an edit to your mod, find the post in the Discussion section.** You can add Hype to the beatmap by submitting Praise in the General (All Difficulties) section.
+
+### Pin
+
+A blue button with a pin icon. Appears to do nothing at this time.
```

Comment:
at the time

Reply:
could you explain why it should be `at the time` instead of `at this time`?

Reply:
It currently appears to do nothing. 
Now it might be better